resources/views/edit/editConfirm.blade.php

```diff
@@ -0,0 +1,24 @@
+<h2>問題・答え編集確認画面</h2>
+<form action="editdb" method="POST">
+@csrf
+<label for="question_id">問題：</label>
+<input type= "text" name= "questions_id" value ="{{ $questions_id }}" readonly />
+<input type= "text" name= "question" value ="{{ $question }}" />
+<br>
+@if(isset($answer_ids))
+@foreach($answer_ids as $value)
+@if($loop->count)
+<label>答え：</label>
+  <input type="text" name="answer_ids[]" value="{{ $value-> answer_ids }}" readonly />
```

Comment:
$answer_idsはオブジェクトの配列ですか？
answer_idの配列かと思ってましたが…もしそうなら
```suggestion
  <input type="text" name="answer_ids[]" value="{{ $value }}" readonly />
```
こうなりますね。
$value値を出力してみた方が良さそうですね。